packages/xod-client/src/messages/components/SnackBarMessage.jsx

```diff
@@ -31,6 +31,38 @@ class SnackBarMessage extends React.Component {
     this.props.onCloseMessage(this.props.message.id);
   }
 
+  getMessagePath() {
```

Comment:
I think you’re messing terminology a bit. The “path” term is used widely in XOD as a notion of owner → libname → patchname. You mean another dimension here. I suggest sticking to a new term here. “Trace” is a good variant you’ve suggested.